src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -32,12 +32,8 @@ const NavHeader = (props) => {
     initialData,
     opacityHeaderAnimation,
     onShareCommunity,
-    getTopicDetail,
-    setMemberCount,
-    memberCount,
     isHeaderHide,
     topicDetail,
-    setIsFollow,
     isFollow,
     hasSearch,
     searchText,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [32-51]

The props received by the component have been modified. The `getTopicDetail`, `setMemberCount`, `memberCount`, `setIsFollow`, and `getSearchLayout` props have been removed, and `followType` and `onFollowButtonPress` have been added. Ensure that these changes do not break any functionality where this component is used.
```diff
-    getTopicDetail,
-    setMemberCount,
-    memberCount,
-    setIsFollow,
-    getSearchLayout,
+    onFollowButtonPress,
+    followType
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->